apps/fabric-examples/src/pages/ButtonPage/examples/Button.Primary.Example.tsx

```diff
@@ -16,7 +16,8 @@ export class ButtonPrimaryExample extends React.Component<IButtonProps, {}> {
         <PrimaryButton
           data-automation-id='test'
           disabled={ disabled }
-        >Create account</PrimaryButton>
+          text="Create account"
+        >Text Overrides a Child String</PrimaryButton>
```

Comment:
I don't think it's a great example, though I can sympathize with having examples to prove scenarios... But not a good one to follow ;)